codec-dns/src/test/java/io/netty/handler/codec/dns/DnsQueryTest.java

```diff
@@ -68,7 +68,10 @@ public void testEncodeAndDecodeQuery() {
             DatagramPacket packet = writeChannel.readOutbound();
             assertTrue(packet.content().isReadable());
             assertTrue(readChannel.writeInbound(packet));
-            assertEquals(query, readChannel.readInbound());
+            DnsQuery decodedDnsQuery = readChannel.readInbound();
+            assertEquals(query, decodedDnsQuery);
+            assertTrue(decodedDnsQuery.release());
```

Comment:
@amizurov can you open an extra PR for this change ?

Reply:
Sure